### ISSUE TITLE
NoMethodError: undefined method `with_indifferent_access' for nil:NilClass

### ISSUE THREAD

**Issue by adrienpoly** [allmarkedup/lookbook#42]
Me again 

I wanted to test if my components would display correctly in staging (testing for #8) and I got this message during the deploy phase on Heroku.

```
rake aborted!
remote:        NoMethodError: undefined method `with_indifferent_access' for nil:NilClass
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/lookbook-0.2.2/lib/lookbook/engine.rb:40:in `block in <class:Engine>'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/initializable.rb:32:in `instance_exec'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/initializable.rb:32:in `run'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/initializable.rb:61:in `block in run_initializers'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/initializable.rb:60:in `run_initializers'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/application.rb:391:in `initialize!'
remote:        /tmp/build_af8e446f/config/environment.rb:7:in `<main>'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.4.1/lib/active_support/dependencies.rb:332:in `block in require'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.4.1/lib/active_support/dependencies.rb:299:in `load_dependency'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.4.1/lib/active_support/dependencies.rb:332:in `require'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/application.rb:367:in `require_environment!'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/railties-6.1.4.1/lib/rails/application.rb:533:in `block in run_tasks_blocks'
remote:        /tmp/build_af8e446f/vendor/bundle/ruby/2.7.0/gems/sprockets-rails-3.2.2/lib/sprockets/rails/task.rb:61:in `block (2 levels) in define'
remote:        Tasks: TOP => environment
remote:        (See full trace by running task with --trace)

```

**Comment by allmarkedup** [comment]
Hey again @adrienpoly :-)

Thanks for the report - I've just released [v0.2.3](https://github.com/allmarkedup/lookbook/releases/tag/v0.2.3) that should fix this error for you. Let me know if that solves it!

**Comment by adrienpoly** [comment]
it does solve the issue thanks
